src/Calculator2.java

```diff
@@ -17,6 +17,7 @@ public static void main(String[] args){
             System.out.println("Wynik dzielenia: " + (firstNumber/secondNumber));
         } else {
             System.out.println("Cannot be divided by zero");
+            
 
 
```

Comment:
These empty lines are redundant, should be deleted. Try to play around with "Ctrl+Alt+L" shortcut in IntelliJ to format source code automatically.
You can also find more useful shortcuts in the [IntelliJ keymap](https://resources.jetbrains.com/storage/products/intellij-idea/docs/IntelliJIDEA_ReferenceCard.pdf).